exercises/Euclidean/python/main.py

```diff
@@ -0,0 +1,65 @@
+def get_nums():
+    while(True):
+        try:
+            num1 = int(input("Please enter a positive integer: "))
+            break
+        except TypeError:
+            pass
+    while(True):
+        try:
+            num2 = int(input("Please enter a positive integer: "))
+            break
+        except TypeError:
+            pass
+
+    return [num1,num2]
+
+def euclid_loop(x,y,prev):
+    big = max(x, y)
+    small = min(x, y)
+
+    r = big % small
+    return prev if r == 0 else euclid_loop(r, small, r)
+
+def euclid(x,y):
+    return euclid_loop(x,y,min(x,y))
+
+def modular_inverse(num1,mod):
+    ret = modular_inverse_loop(num1 % mod, mod, mod, 1, None, None, None, None)
+    if (ret == False):
+        return str(num1) + " does not have an inverse mod " + str(mod)
+    else:
+        return "The inverse of " + str(num1) + " mod " + str(mod) + " is " + str(ret)
```

Comment:
functions should return the actual numbers (with a comment explaining what is returned), and the caller of the function should be the one to determine how it is used or put into a string